select-xamarin-sdk-v2.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e -o pipefail
+
+echo "Set SDK"
+
+get_framework_path() {
+  local framework=$1
```

Comment:
nitpick: Considering simplicity of function, do we really need a separate variable?